Email_Scheduler.py

```diff
@@ -0,0 +1,56 @@
+import smtplib, ssl
```

Comment:
A README.md file, with a screenshot embedded for the program would be a good add on, for it to be a masterpiece. I would say.

Reply:
Embedded screenshots would like require me to reveal my email address, which I would not be very comfortable in. 
I can add a README.md for this, though I have added comments to this code.

Reply:
Yeah, README.md would be fine.